website_anchor_smooth_scroll/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Smooth Scroll for Website Anchors',
+    'summary': '''Replace default behavior for internal links (anchor only)
```

Comment:
this is going to add a carriage return into the summary. Instead you can break the string, then add a new one below it and it will auto-concat

```
'Replace default behavior for internal links (anchor only)'
'with smooth scroll'
```